config/StoredProcedures/GetAllInventory.sql

```diff
@@ -30,4 +31,4 @@ on s1.ProductID = s2.ProductID;
 
 END $$
 
-DELIMITER ;
\ No newline at end of file
+DELIMITER ;
```

Comment:
I like the IFNULL first of all! The correctness of this solution might be db engine specific. Since we know we are always going to use mysql, it's good enough. Let's use this.